README.rst

```diff
@@ -128,19 +174,30 @@ This section outlines what stages a proposal may go through. The stage is identi
    * `≡ List of proposals being revised <https://github.com/ghc-proposals/ghc-proposals/pulls?q=label%3A%22Needs+revision%22>`__
    * `≡ List of rejected proposals <https://github.com/ghc-proposals/ghc-proposals/pulls?q=label%3A%Rejected%22>`__
 
-7. **Label**: `Dormant <https://github.com/ghc-proposals/ghc-proposals/pulls?q=is%3Aopen+is%3Apr+label%3A%22Dormant%22>`__.  If a proposal sees no activity for along time, it is marked as “dormant”,
-   and eventually closed.
+8. **Label**: `Pending committee review <https://github.com/ghc-proposals/ghc-proposals/pulls?q=is%3Aopen+is%3Apr+label%3A%22Pending+committee+review%22>`__.  
+   **Responsibility for next action:** shepherd and author.  **Timescale**: two weeks. 
+  
+   If the committee accepts the proposal with minor corrections, the author and shepherd work together to make any final edits.
+
+9. **Label**: `Accepted <https://github.com/ghc-proposals/ghc-proposals/pulls?q=label%3A%22Accepted%22>`__.  
+   **Responsibility for next action:** shepherd and committee secretary.  **Timescale**: one week. 
+
+   If the committee fully accepts the proposal, the committee secretary and/or shepherd marks the proposal as “accepted” and merges the Proposal.
+   Any later changes must be submitted as a separate amendment proposal.
+
+10. **Label**: `Implemented <https://github.com/ghc-proposals/ghc-proposals/pulls?q=is%3Apr+label%3A%22Implemented%22>`__.   Once a proposal is accepted, it still has to be implemented.  The author
+    may do that, or someone else. We mark the proposal as “implemented” once it
```

Comment:
I think it's worth being clear that acceptance of a proposal does not (unfortunately) guarantee someone will step up to implement it.

```suggestion
    may do that, or ask someone else to do so. GHC is a volunteer-driven project, so there is unfortunately no guarantee that accepted proposals will be implemented promptly.
    
    We mark the proposal as “implemented” once it
```